contracts/InterchainTokenFactory.sol

```diff
@@ -175,27 +174,19 @@ contract InterchainTokenFactory is IInterchainTokenFactory, ITokenManagerType, M
      * @return tokenId The tokenId corresponding to the deployed InterchainToken.
      */
     function deployRemoteInterchainToken(
-        string calldata originalChainName,
         bytes32 salt,
         address minter,
         string memory destinationChain,
         uint256 gasValue
-    ) external payable returns (bytes32 tokenId) {
+    ) public payable returns (bytes32 tokenId) {
         string memory tokenName;
         string memory tokenSymbol;
         uint8 tokenDecimals;
         bytes memory minter_ = new bytes(0);
 
         {
```

Comment:
wondering if this additional scoping is still needed now that we removed one variable (maybe stack too deep error goes away)

Reply:
That's a great call, actually it works without additional scoping. Updated.